Server/JokesOnYou.Web.Api/Repositories/UserRepository.cs

```diff
@@ -7,22 +7,37 @@
 using JokesOnYou.Web.Api.Repositories.Interfaces;
 using Microsoft.AspNetCore.Identity;
 using Microsoft.EntityFrameworkCore;
+using System;
+using System.Linq;
+using AutoMapper;
+using JokesOnYou.Web.Api.Exceptions;
 
 namespace JokesOnYou.Web.Api.Repositories
 {
     public class UserRepository : IUserRepository
     {
         private readonly UserManager<User> _userManager;
+        readonly IMapper _mapper;
 
-        public UserRepository(UserManager<User> userManager)
+        public UserRepository(UserManager<User> userManager, IMapper mapper)
         {
+            _mapper = mapper;
             _userManager = userManager;
         }
 
         public async Task<User> CreateUserAsync(UserRegisterDTO userRegisterDTO)
         {
-            var userCreationResult = await _userManager.CreateAsync(new User() { Email = userRegisterDTO.Email, UserName = userRegisterDTO.Email }, userRegisterDTO.Password);
+            /* Just in case we need it
+            if (_userManager.Users.Any(x => x.Email == userRegisterDTO.Email))
+                throw new AppException($"Email {userRegisterDTO.Email} is already taken");
+
+            
+            if (_userManager.Users.Any(x => x.UserName == userRegisterDTO.UserName))
+                throw new AppException($"Username: {userRegisterDTO.UserName} is already taken");
+            */
 
+            var userCreationResult = await _userManager.CreateAsync(new User() { Email = userRegisterDTO.Email,
```

Comment:
Maybe use mapping for this ?
If we add any new fields in the UserCreateDto that will automatically be added. not sure about how password will be handled if you use automapper but you might be able to exclude it?
since the name in the Database is different it should not include it anyway.

Reply:
I don't think a map is needed for this, 
@chrisK00 do we need to map this?